### PR TITLE
Update credits for 2.1

### PR DESCRIPTION
**Pull request by ghost**
Signed-off-by: ziycon ziycon.eire@gmail.com

### CODE REVIEW

**Comment by illori** [comment]
actually there is no former developer section as that is a comment that is removed.

also josh should stay as a member in the list as he was a developer while 2.1 was in development that has always been the policy

also per the comment remove spuds as he does not want to be listed.

also if you read the PR would you see the leads are no longer in the list separate so it would not matter if it is merged now or later.

**Comment by norv** [comment]
There are several mistakes in the PR and comments.
I have submitted corrections to @ziycon branch: https://github.com/ziycon/SMF2.1/pull/1

**Comment by silentguardian** [comment]
The contributions of the people under the updated developer group (in the original pull request) to the current 2.1 codebase is nothing compared to the dozen you have removed. There are a few out there that seems to have gotten in there mistakenly as well, unless making a few little changes here and there makes you a developer. And although I did not want any recognition on the sm.org site, I want my name to be kept in the software credits as long as those who worked on getting 2.0 out are listed there.

**Comment by norv** [comment]
A couple more corrections. My apologies for being still on the run. I shouldn't be, but it is how it is. Just FYI.
Fixing Labradoodle customizer credits:
https://github.com/norv/SMF-old/commit/46c9f2f80d7e76137972fd90706378e7c73f13c5
Re-adding MissAllSunday:
https://github.com/norv/SMF-old/commit/ba79e485c496a53be4bb7f60b545fe0670db3a4e

For all issues, please see the commits messages.

In addition. As I said in the PR above, I did not even try to address the rest of team credits. But I have to note.
The PR has removed also credits for people like LexArma, who has supported this project up to its last few months. The disrespect and mentality of "they're just friends so ignore them for 'our' project" has long taken over around there. Not surprising, and not fixable. But it needs said.

I am rebasing this stuff and fix credits, FWIW, to a version as good as possible, to my present knowledge.

**Comment by MissAllSunday** [comment]
.______.

 It seems you're re-adding my by force or rather forced... since you don't even bother to put the correct name:  Jessica &quot;Suki&quot; Gonz&aacute;lez (github automatically converts the acute entity but it should be &a acute;)  when you attempted to fix the original PR :(    Anyway, it feels forced... it feels I was added back because of my complains, not because of my contributions to the project. Which really is a shame.

BTW, even if you don't want to credit me as a team member during the 2.1 development cycle, I did contribute and as a result I have the legal right to be credited on this mythological new file for contributors you said you will include (and so far I haven't see it). Again is a shame you have to put me in because you are legally forced instead of give me proper credit because you acknowledge me.

**Comment by Spuds** [comment]
> also per the comment remove spuds as he does not want to be listed.

Just to clarify, I had that added as the "process" of who, where and why people were listed was (and apparently still is) completely broken.  

There were simply people in wrong areas and some missing altogether, etc, etc ... it had just become another disgraceful item in SMF's history. So I did not want to be added in anywhere until we knew what we were doing, and applied those rules equally to all.

**Comment by norv** [comment]
I believe I submitted MAS name in my own hotfix linked here. (which used to be her choice).

So, you're not going to answer? So you're just going to 'fix' by just ignoring my explanations and comments, and thereby add the most shameless commits into this repository history?
To think I've been even patient with this crap when I saw it (because I can think of momentary lapses or mistakes, and anyway I'm not blaming individual new team members), but you're just ignoring my explanations? You - the privileged closed-groups backing up this PR, don't care how you call yourselves, or in the name of what.

Shame on you!

Shame on anyone who even thinks to commit such bullshit! You're removing Compuart, Grudge, Aaron, etc, from credits. Then ignore my -too- polite warnings on your lack of clue and responsibility that's driving such action, and 'just thought to add some previous developers' to names with no contribution, no explanation, nothing. You're about to make the most shameless commits to be stored publicly in a broken project repository history.

Enough is enough. Close this PR.

This repository has more than [enough](https://github.com/SimpleMachines/SMF2.1/pull/239) and [even more than enough](https://github.com/SimpleMachines/SMF2.1/pull/303) already!
But THIS is more than unacceptable.

Do you seriously play around with the footprint left on the project by these people?
Do you "make decisions" whether to 'add some previous', huh! To what, to thirty-nothing, and [careless](https://github.com/SimpleMachines/SMF2.1/pull/348) at that?
This project has been driven by _these_ developers. It's their work that has moved this project forward. Forward, not nowhere. It's their work that has gathered around it a community, which includes _you_ as well. It's embedded in the software, in the direction of the code, in the nights they spent for the best solutions for years, in the lessons and guidelines an entire community of modders and themers has been following!

Make sure you go on chitty-chatting about the Core Values of this poor broken project, and look at yourselves. Take a good hard look at these commits and this repository history. 10 years of it! If you don't know what you're doing, _you don't touch them!_

TL;DR:
- Developers _are the previous developers_. That's the flavor they come in.

Close this shameless PR. "SM/SMF" is unfixable under this name/groups/history, but as long as this broken project repository still stays online, I'm NOT going to sit by and watch this.

Close the PR of shame! _Sooner, please._

**Comment by Kindred-999** [comment]
Norv,
Whether you agree or not, enough is enough. Knock it off.
All contributors, whether they be current or former team, smf friends, or outsider contibutors will be recognized.
No one's name is being removed from the cotnributors list and no developer, current or former is losing any credit. If we choose to change the format of the "current" credits and display former developers on the contibutors list, then it is no business of yours - all credits still remain to former contributors, even if it's not the way you "want".

The exact format and method of display for the credits page is still being considered...   however you are not doing your argument any good by attacking everyone and everything in this manner.

**Comment by MissAllSunday** [comment]
I don't really get it, did you or did you not changed your politic about credit every team member present during development cycle?

If you indeed did it, why not been open about it? why not release an announcement clearly stating this change?

Didn't you said to me you were going to be more open?  is this your definition of openness? making a substantial  change in an obscure way, a change that even clashes with your every own [core values](http://www.simplemachines.org/about/values.php)?

> We will recognize the successes and accomplishments of the team and the individuals who make up this team.
> We will give credit where credit is due. This applies to successes, contributions, licenses and rewards.

Is not the way you want either,  for developers you have to honor the license you chose for your project. There is no previous or current devs there is no pals/enemies, there are devs period, and all of them has a copyright and the right to be credited in a dignified way.

As for former team members you have all the rights to do whatever you want and remove however you see fit, I will plea to your sense of dignity and gratefulness towards all the people who contributed something to your project but thats a lose cause since you don't have any sense of dignity or gratefulness.

It may seem logical (and it is for everyone else!) to acknowledge those who stood by your side and help you build the project you're so jealously try to keep it to your self, its a matter of principles really, a matter of the most basic sense of morality.

Even if you no longer have principles to attach to, at least have the courtesy to inform everyone else of your shameless actions, don't take the coward way and hide your actions behind an obscure PR, take some pride on your actions god dammed!

**Comment by Kindred-999** [comment]
Hi Jessica,

I am uncertain what you are on about.
We fully intend to give credit to _ALL_ contributors, current and former alike.
Additionally, what is there to "announce"? First of all, this is a pull request, not a commit. Second, even if it was a commit, why would we need to "inform everyone"? We don't "inform everyone" with some sort of public blast for every pull or commit made, why would this be any different?
Regardless of that...  You are getting worked up over Norv's BS. As I have said - everyone, current and former, will be credited. That is rather the point of the whole contributor's file isn't it?
Oh, and there are current and former devs - there are devs who have contributed specifically during the current development phase and there are devs who have contributed to prior developments. If they did not contribute during the current phase, but still have code in the repository, then they are former devs...  doesn't mean anything except when they contributed.

**Comment by norv** [comment]
> If we choose to change the format of the "current" credits and display former developers on the contibutors list, then it is no business of yours - all credits still remain to former contributors, even if it's not the way you "want".

Excuse me, WHAT?
So the privileged closed-doors "we" seriously "chooses" to remove developers credits?

Let me say it in plain English: No.

Don't you dare!

Whoever chooses to be personally offended by the truth, suit yourself. You get (less than) the reaction the above shameless words deserve.

Correct and fair acknowledgement only comes with care _for everyone_ in _the right proportion_ .
Everyone means everyone, the community.
Proportion means based on the MERIT of their work, on what you DO, on the relevance for the project.

These "former" developers have moved this project forward, and their work does so today as well.
Don't you dare remove developers credits from the software!

> we choose

_You_ "choose" who did what for this project?
No. You do not.

You "choose" to remove Compuart's credits from the software, and confine them to a text file uniformly listed with thirty-something commits?

DON'T YOU DARE.

No. Is that English enough for you?

> we choose

I know all too well your "we", the SM/SMF 'team'/'corporation' structures, are essentially hopeless, don't worry, I am not blaming you or anyone else for that hopelessness. Not even you, Kindred, even though you know and I know that you have done everything and anything, to raise every ego of those who don't do the work against those who do. Until no one - no developer, and thereby no one - can stand to what has become in there.
No. I am not even blaming you, Kindred, whether you believe or not or what you twist it into is irrelevant.
I am blaming only **myself** for that.

But. Don't you dare remove developers credits from the software!

As long as they are developers of this software, and some sort-of-kind-of distribution somewhere under this name, shame on you for even considering it.

---

**Developers of 2.0 are developers of 2.1.** Spuds and Ema are developers of 2.1.
There are no 'former' developers = all developers are 'former' = anyone who tries to pick up is great from their part, just not proven yet.
It's that simple. Hopeless for many reasons, but simple. Which part of it you don't get?

SMF-2.1-dev codebase is 2.0 with many fixes, very localized reworkings, some new features, some theme additions, some tweaks. Numbers say something but don't entirely do it justice, generally Spuds works much more than anyone else before he commits once, OTOH some of my later work has been recommitted by him with no author info.
I know. You do not.

I'm not entirely against crediting as developers a few of the new developers (although some contributions are too disproportionate as yet and/or not to 2.1-dev): if and only if they and the (very small) development community around here is ok with it. It is not my first choice, as you can see in my hotfixes, but it's up to them to make it a non-issue.

I have entirely and intently decided to credit normally the last developers of 2.0 in 2.0 because it has been well deserved.
Yes, I absolutely consider they were developers of 2.0. (this includes Nao, playing groups is irrelevant.)
I did not and do not even consider to remove developers credits from the software, for all versions post-2.0 as yet. (might've for 3.0, but even then, major contributors would've remained for several reasons.)

> Oh, and there are current and former devs - there are devs who have contributed specifically during the current development phase and there are devs who have contributed to prior developments. If they did not contribute during the current phase, but still have code in the repository, then they are former devs... doesn't mean anything except when they contributed.

No. Which part of No you don't get?

Don't you dare remove Compuart's credits from this software!

For developers, THEIR WORK is all essentials of the core software  _of 2.1-dev minor version_. Just in case there'd be any more distribution, and in the history of this repository:
Don't. You. Dare.

> we choose

No. You do not.

I hate repeating myself, but: close the PR of shame.

**Comment by Kindred-999** [comment]
more FUD...  hardly "truth".

which part of "no longer actively participating" != former?  Isn't that the very definition of "former"?  Yes, I do believe that it is.

**Comment by LiroyvH** [comment]
Norv,

If you have absolutely no idea what you're talking about, how things are done and who is in charge of what or "forces" things (as you are clearly oblivious to facts even though you call your unfounded opinions "facts" (which they are not.)): please simply remain silent and stop poisoning people with your constant nonsense that is based on absolutely nothing at all. Thank you in advance.

Everyone will be credited where credit is due as that's what we believe in and even if we didn't believe in it: that's how it should be done non the less. We appreciate everyones contributions _very_ much and the absolute least we can do is credit them. I'm sad to see people think we don't want to credit contributors, it is absolutely ridiculous. Sorry for the wording. :)

This is A WORK IN PROGRESS. Nothing is definite. Nothing is merged. We're still in the discussion phase on how to do it, and I don't even know why I'm telling you this as you have your own project to worry about and what we're looking in to doing and what we're testing is absolutely not your concern or something to meddle with.
We are looking in to it and how to do it. While that is in progress, please stop sprouting your nonsense and poison around and just wait till we release a final version.

If you still have complaints about the final version: fine.
But now, to repeat Kindred's words: knock it off, please. 

Thank you in advance and have a great day.

**Comment by MissAllSunday** [comment]
Kindred, you haven't answer my question, did you or did you not changed your politic regarding giving credits to all team members present during a development release?

If you're indeed going to change your policy, don't you think you need to make an announcement? after all, that policy has been around since very early days, it is part of your core values and it affects former team members, again, it would be nice of your part to make such a gesture but I see you just don't care about former contributors.

Please do not mix up things here, one thing is developers who by the license you chose have the right to be credited and another totally separated thing is the former team members who can't legally enforce their right to be credited in a dignified way.

I care about the later, there is no legal ground for former team members to retain their credits since their contributions cannot be measured neither can be agglutinated under the BSD license, stuff like customizations for example, support, i18n stuff, etc, cannot be covered by the current license thus theres a limbo here and our earned right to be credited only depends on your own good will, which I gotta say, you don't seem to have. Thats my concern, you don't seem to be interested on keep the current policy.

The devs thing is a no brainier, legally they deserve to be credited, there is nothing you can't do to stop it, they need to be credited as you're using their code on your own project, this is standard OS behavior. You cannot mess this up, legally you can't.

 Oh well, I truly hope whoever is in charge by the time you're a former team member doesn't have the same aggressive state of mind and I truly hope you will get credited for your efforts, no one deserves to be left out, not even the ones who are trying to enforce such mentality.

**Comment by colinschoen** [comment]
Excuse me for interrupting, but what is so unclear?

"We fully intend to give credit to ALL contributors, current and former alike."
"All contributors, whether they be current or former team, smf friends, or outsider contibutors will be recognized."
"No one's name is being removed from the cotnributors list and no developer, current or former is losing any credit."

How many more times can he repeat the same thing and you personally attack him? 

"I will plea to your sense of dignity and gratefulness towards all the people who contributed something to your project but thats a lose cause since you don't have any sense of dignity or gratefulness."

That is not appropriate nor acceptable.

**Comment by MissAllSunday** [comment]
First of all, thanks! it is indeed very refreshing to see new faces responding.

No I'm not attacking anyone. I'm expressing my opinion regarding your actions, thats about it.

You (plural) said you have full intention of give credits where credit is due, however this  mere PR says otherwise, this PR is an action, a solid action that may be implemented on the final version, your words or rather text is just that, words.

Actions are the ones who matters not words, so, you can say it as pretty as you want, you can say it as many times as you want, what really matters are actions and right now your action is to remove former contributors.

Please don't say words, do actions instead!  shut my mouth with actions! not words!   I certainly cannot bitch about actions, I cannot argue against actions so, if you want me to shut up, what better way to do it than take proper action!

**Comment by colinschoen** [comment]
"I will plea to your sense of dignity and gratefulness towards all the people who contributed something to your project but thats a lose cause since you don't have any sense of dignity or gratefulness."

Unfortunately, that is a personal attack. When you directly address someone and state that they don't have any sense of dignity or gratefulness you are attacking their character and integrity. If you truly, truly want to accomplish something you would keep the discourse civil. As I am sure you see by this point, getting emotional and personal doesn't achieve anything.

We haven't taken any action and any action we take will not remove credit associated with former contributors, as previously stated. A pull request was made to create a credits text file. You said "right now your action is to remove former contributors", but that was not what was proposed. It was to move the credit into a text file. Now, you need to understand, the notion was not accepted or made official by being merged into the main repository. Our opinions are voiced and now we should move on.

**Comment by MissAllSunday** [comment]
Like I said, it was an opinion towards an action and the "you" was mean to be plural as in you guys the whole team going in that direction. There was nothing emotional or personal there, it was an opinion based on your actions, if you see that as an attack so be it, I don't need to convince you otherwise.

I cannot see any PR to create a credits file, all I see is a PR removing contributors.  In this PR there is no indication of the creation of said file, no one mentioned said credits file before, you are the first one to even mentioned that file. So no, trying to make this PR look innocent just isn't working, thanks all mighty Poseidon for git fanaticism of loggin everything as everyone has access to the original PR where there was nothing but removals of contributors and absolutely no mention of the creation of a credits file.

Take into account this is an actual PR, a PR in git world is something you intend to be part of a project, usually you send complete PRs, in this case if the intention truly was to create a text file, there would have been more commits creating said, currently there is no indication of said intention.

So please allow me to respectfully disagree with you, there is/was absolutely no intention of creating said credits file and I will also encourage you to drop that argument in its entire as it does nothing but insult the intelligence of those concerned with the intention of this PR. Thanks.

**Comment by Spuds** [comment]
I believe what you are hearing is that some feel its inappropriate to remove 2.0 developers and relegate them to a "readme" file in the disto.

Really 2.1 is for the most part a massive bug fix with some lipstick on the pig (sorry if anyone finds that offensive, but hey I added a lot of that lipstick lol)  

So (to me) the consternation is, should developers like Compuart, Grudge, Aaron, etc be replaced in the main credits with the names of bug fixers and css tweakers?  Personally I don't think so, when I look at what I did on 2.1, I know the totality of my efforts does not hold a candle to what they and others did for SMF 2.x

The fact that SMF still runs as well as it does, in spite of it not getting any proper care and feeding in years, is a testament to the work of those being moved to the "archive" credits (so to speak).  So I'd ask others to reflect on if they feel that is an appropriate action (setting aside for the moment whatever the "procedure/process" was/is/has been etc

**Comment by colinschoen** [comment]
@MissAllSunday - I think my response to Spuds will also be applicable to what I intended to respond to your note.

Thank you Spuds.

Now that is the substance of the issue I was digging for. I can't agree more with your statement of the quality work many previous developers committed to SMF and the strong foundation they left for the future of the software. Maybe a solution would be renaming the titles listed on the credits page. How about something like: Recent Developers and Recent Localization contributors. How do you feel about something like that?

**Comment by MissAllSunday** [comment]
@colinschoen thank you for your concern and your initiative of requesting input from external sources, this is indeed refreshing, I just hope you're not the only one who have this initiative nor you're saying this just to get off the hook, you won't let me lie if I say both coreisp and Kindred's comments on this issue doesn't really shows the same initiative you're showing. Thats nothing new, this is why someone else responding is so refreshing. It brings other perspectives equally valid, I hope more team members get involve and participate and finally stop letting the same persons be the face and voice of the project in every "controversy" and I hope more team members do realize this is not just drama. The more people participating the better.

**Comment by colinschoen** [comment]
Thank you @MissAllSunday. I can assure you that I am being genuinely sincere.

What are your thoughts about amending the titles, proposed above?

-- I also forgot to clarify earlier, there indeed is intended to be a contributors list included in the repo as I initially mentioned. That wasn't clearly, initially stated which is a fault and possibly a large factor of the controversy. Sorry about that. The only thing I can say is that it was and is something that many concurred on. If warranted, someone else can similarly attest to that.

**Comment by norv** [comment]
Allow me to get this straight:
A couple of people who never contribute to this software have invented a "problem", and now waste everyone's time and energy to "look for solutions" in the wrong place. With a single result: remove, minimize and uniformize even _credit where credit is DUE_.

My answer is: No.
There is no "solution" to look for, since there is no problem to begin with. There are no 'former developers' when it comes to the software. _In particular_ for the project under this name.
There have not been in 2.0. There is no "reason" in hell to invent so-called 'teams' for a _minor_ formerly intended milestone, what we call 2.1-dev in this repo.

But, if you really want to 'split' developers, lets keep this constructive, and here's a way: 
- core developers, all pre-2010, except Orstio.
- developers, the rest in my PR.
- contributors, everyone who has contributed to the software.

It is possible to have a couple as core devs from post-2010.

And if you want to talk about 'former developers': then amend the page to state clearly
"SMF Developers are former developers."

@colinschoen : I do not do personal attacks. And I am not blaming you or new team members for falling prey to this insanity.

@ziycon : You don't want to be part of this. If they're so sure of themselves, let _them_ try it.

@Oldiesmann : Close the PR of shame.

**Comment by colinschoen** [comment]
Thanks Norv,

That was very useful. I quite like what you did with the developer groups. I think that might be a great start. What about now separating the developer group you listed into current developers and former developers? I think this would still align with the goal of rebuilding the credits while awarding proper credit. I guess the question now is splitting the current members and other groups into the two areas. With the following titles, how do you feel about the secondary file containing the former members and contributors?

I don't think this is insanity. I think it is good that we are getting to a point where we can civilly go back and forth with parallel ideas and suggestions.

**Comment by MissAllSunday** [comment]
@colinschoen  First of all, thank you, and again let me say you how incredible refreshing it is for a team member to acknowledge there was a fault somewhere, we are all humans and made mistakes but somehow the team has had a long history of denying or hiding any possible fault they might had committed, I tried to address that by been open and sincere with all the actions I took during my time as a team member, this is truly remarkable and I truly hope more members can follow your example.

###### About devs

I don't know about developers, that is a whole different matter and it must be carefully addressed since theres legal matter involved, INAL and I do not have access (nor should I) to your talks with the Software Freedom Law Center so I won't even comment on this matter.

###### For former team members

 I would remove any "labels" meaning there shouldn't be any separation between the different subgroups your project has, many team members has changed between sub teams many times making it difficult to know to which team x member should be credited. I concur sub teams have a purpose and a meaning in an organization, however, this is the credits page, there is no need to keep such strict organization when you can simply give credit to every single person who is/was a team member during X release development cycle.

I wouldn't touch the current credits page, if anything only to remove the subteams separation, this gives equality to everyone, current and former team members, make a nice list, in alphabetical order, listing every single person who was a team member during development cycle, This creates equality, shows unity towards a common goal.

Please do note devs are a special case here since by law they should be credited even if there weren't present during the development cycle, this is because devs gets credited by code and code is the only thing that gets covered by the license thus devs have its own set of rules and must be treated separately and much more carefully.

**Comment by LiroyvH** [comment]
Ignoring @Norv's nonsense troll comment;

Jessica, I have to correct one thing; there was no direct fault. That's not an attempt to hide anything or "not admit" anything, it's simply the truth. :) Let me explain:

This is simply a PR to check things out, nothing is even merged yet and it's simply being looked at. 
That it gets ripped out of context is too bad but seems to be the way things go over and over.
That this particular PR may contain flaws can certainly be the case and I am **NOT** denying that! Not at all! But to immediately say it's horrible "this was done" is unfounded imho, nothing IS done yet... It's a mere starting point, and more will probably follow that may also contain flaws... Please keep that in mind. It's not always easy to get everything 100% right in one go. :) Sorry to see it was instantly used for mud throwing.

That's why it's so saddening to see that somehow people feel there's this need to immediately rip it out of context and start saying how evil people are. Not to mention the complete nonsense Norv is uttering regarding people and who does what while she hasn't got a single clue who is responsible for what, who decides to do things, et cetera.
But I'm kinda used to that now.

Had to clarify that and I'm sorry to see you misinterpreted the whole ordeal, but I cannot say I blame you.

Thank you VERY much for providing suggestions, that was actually more the point of this PR... Test and gather input for the discussion. (Note: not specifically from you, but you are more than welcome to :)) Not test and instantly merge unfinished business.

Glad to see it finally got on track. :) Thank you for helping with that and providing suggestions and solutions. :)
It is truly appreciated.

-edit-
Oh and yeah, trust me, I don't comment here for my pleasure or on other issues. Simply am forced to due to the fact that initially no one else responds and we have to ensure that the slander is put to a halt.
In the future, it may work better if people ASK first what the idea is, rather than instantly start accusing and pointing fingers. :) That way, you may never hear from me ever again and live up to your preference. :)

**Comment by Kindred-999** [comment]
I am glad we are back on track with this, but I do want to point out one thing....
You mention: "... no one mentioned said credits file before...
but I did.... in my first response to you: "That is rather the point of the whole contributor's file isn't it"

**Comment by MissAllSunday** [comment]
It may be even better to STATE first whats the idea behind a PR, specially if you're planing on develop this idea on separate stages. Standard behavior when working under a team environment.

I did not misinterpreted anything, I saw an action, the original PR, there cannot be any misinterpretation when the single action that was given is removing contributors. Sorry but I had to clarify that. Anyway, you "partially" admitting there was a flaw can be considered as progress. 

Again, you do not simply send a PR for the sake of sending a PR, ideas are developed on local repos and they are sent when they are finished, thats why local repos do exists and thats exactly how your team has been working so far. This is crucial in team development, developing an idea in staged chunks is counterproductive towards development itself.

Anyway, I truly hope more team members to dare to express their equally valid opinions, it can only serve to good things having a different set of mind to what I'm used to whenever I raise some concern.

@Kindred-999 I now you did and I amde that statement fully knowing that you did.

That is yet another thing you haven't clarify...  like I already said multiple times, theres two separate issues here, one is the devs, former devs and contributors and another one is former team members.

The contributors file you did mentioned is a legal requirement you guys needs to comply, this contributors file only covers devs, former devs and people who contributed code, this contributors file only covers the code aspect of the project. The code aspect of the project is the only thing that is covered by the license and the only thing you're legally required to fulfill.

The "credits" file is another thing altogether, I surely hope you aren't planning on stuffing all former team members on a separate file and only keeping the "current" ones on the credit page, thats not a sign of good will towards former team members, former team members have the same equal right of appearing on the credits page as the current ones.

If you want to keep your separate file, then make it equal by removing the entire credits page and stuff everyone, current and former team members on that credits file, that would show good will towards unity and equality.

**Comment by jdarwood007** [comment]
Seems to be stupid all this work to put people into certain groups and move them around according to their current team status.

Why not make it a few simple groups being SMF developers (main dev team), SMF Team members (the team), Contributors (try our best to note those who made a sizable contribution), Simple Machines NPO members (+BOD) and as always our general thanks to beta testers, charter members, community members and anyone we missed (try not to though).

**Comment by eurich** [comment]
> If you want to keep your separate file, then make it equal by removing the entire credits page and stuff everyone, 
> current and former team members on that credits file, that would show good will towards unity and equality

yep, exactly my feelings.. It's not acceptable for me as it makes current team members more important than the code contributors, but more than 95% of the SMF code is from former developers ;)

btw, the original list of "developers" is still incomplete I still miss a bunch of former ( code & graphics) contributors:
Unknown, Jeff Lewis, Joseph Fung, Jeremy "SleePy" D.,  Matt "SlammedDime" Zuba, Nao, Michael "Oldiesmann" Esholm, Jonathan "vbgamer45" Valtentin, Christian Land, Tim Ceuppens ...

Some never had the "developer" title, but they contributed code and/or graphics and made SMF to what it is...
The old CVS / SVN log should probably show some more, someone needs to check it.

**Comment by norv** [comment]
> Seems to be stupid all this work to put people into certain groups and move them around according to their current team status.
> 
> Why not make it a few simple groups being SMF developers (main dev team), SMF Team members (the team), Contributors (try our best to note those who made a sizable contribution), Simple Machines NPO members (+BOD) and as always our general thanks to beta testers, charter members, community members and anyone we missed (try not to though).

A very good idea. Re-start from here.

**Comment by ghost** [comment]
@norv what you think of http://cus.bryandeakin.com/21/index.php?action=credits

appologies if I missed a dev or two but this is most ive ever spent on a single file just say whos missed and ill add
